sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -169,7 +169,7 @@ class WholeStageCodegenSuite extends QueryTest with SharedSQLContext {
 
       val plan = df.queryExecution.executedPlan
       assert(!plan.find(p =>
-        p.isInstanceOf[WholeStageCodegenExec] &&
+        p.isInstanceOf[WholeStageCodegenExec] && p.isInstanceOf[SortMergeJoinExec] &&
```

Comment:
why this change?

Reply:
now this becomes always false.

Reply:
@cloud-fan Very sorry.. Got checked-in by mistake (I was probably debugging). I will revert.